api

```diff
@@ -330,6 +330,10 @@ The Pi-Apps developers have been receiving a few of these errors recently, but w
   
   #by default, apt-ftparchive will generate lines like "Filename: ./package-name". This seemed to have caused one error-report, so we remove the "./" to hopefully solve the problem.
   sed -i 's+^Filename: \./+Filename: +g' /tmp/pi-apps-local-packages/Packages
+
+  #hash the repository by creating a Release file
+  (apt-ftparchive release . > Release) || error "repo_update(): apt-ftparchive failed to hash the repository: /tmp/pi-apps-local-packages
```

Comment:
As with line 325, I think `cd /tmp/pi-apps-local-packages && ` is necessary.

Reply:
> Isn't it already in that folder from the CD in 325. No need to CD twice

No, because the parenthesis forms a subshell that isolates the internal current directory with the outer one.

Reply:
fixed. I was testing locally and was already in that location so I didn't notice. https://github.com/Botspot/pi-apps/pull/1910/commits/a07d1f8be70de0363d02311f530c6b937d503f90

Reply:
This is exactly what a review process is for! :)
I'll merge this shortly.